providers.tf

```diff
@@ -0,0 +1,14 @@
+provider "aws" {
+  region = var.region
+  assume_role {
+    role_arn = var.firewall_manager_administrator_arn
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
```

Comment:
Modules should not define provider blocks.

https://www.terraform.io/docs/language/modules/develop/providers.html#legacy-shared-modules-with-provider-configurations